packages/beacon-node/src/chain/stateCache/persistentCheckpointsCache.ts

```diff
@@ -544,19 +546,26 @@ export class PersistentCheckpointStateCache implements CheckpointStateCache {
             firstState = state;
           }
 
-          // amongst states of the same epoch, choose the one with the same view of reloadedCp
-          if (
-            reloadedCpSlot < state.slot &&
-            toRootHex(getBlockRootAtSlot(state, reloadedCpSlot)) === reloadedCp.rootHex
-          ) {
-            return state;
+          try {
+            // amongst states of the same epoch, choose the one with the same view of reloadedCp
+            if (
+              reloadedCpSlot < state.slot &&
+              toRootHex(getBlockRootAtSlot(state, reloadedCpSlot)) === reloadedCp.rootHex
+            ) {
+              this.logger.verbose("Reload: use block state as seed state", {stateSlot: state.slot, ...logCtx});
```

Comment:
Looks like the log fix was not applies here https://github.com/ChainSafe/lodestar/pull/7501/commits/2698a51ab632f25a3a76c2477615aac37a05157c

Reply:
@twoeths do we wanna apply the log fix to unstable? just going through changes we have not yet committed there

Reply:
ah good catch thanks @nflaig

Reply:
tracked in https://github.com/ChainSafe/lodestar/pull/7600